superbench/benchmarks/micro_benchmarks/tensorrt_inference_performance.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""TensorRT inference micro-benchmark."""
+
+import re
+from pathlib import Path
+
+import torch.onnx
+import torchvision.models
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Platform, ReturnCode
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class TensorRTInferenceBenchmark(MicroBenchmarkWithInvoke):
+    """TensorRT inference micro-benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'trtexec'
+        self._pytorch_models = [
```

Comment:
I think we can add all models and use config for the model selection.

Reply:
It should include all model benchmark.

Reply:
Is it possible to unify the torchvision models declaration in Model Benchmarks and TensorRTInferenceBenchmark?

Reply:
it can run all models in torchvision, this is just the default models, both are consistent with cnn models in model benchmark